lib/pbench/server/api/resources/__init__.py

```diff
@@ -64,6 +65,24 @@ def __str__(self) -> str:
         return f"{'User ' + self.user.username if self.user else 'Unauthenticated client'} is not authorized to {self.operation.name} a resource owned by {self.owner} with {self.access} access"
 
 
+class UnauthorizedAdminAccess(UnauthorizedAccess):
+    """
+    A refinement of the Unauthorized exception where ADMIN access is required
+    as we can't describe a resource owner and access.
+    """
+
+    def __init__(
+        self,
+        user: Union[User, None],
+        operation: "API_OPERATION",
+        http_status: int = HTTPStatus.FORBIDDEN,
+    ):
+        super().__init__(user, operation, None, None, http_status=http_status)
```

Comment:
Any particular reason why you specified the keyword only for the last parameter (where it looks redundant)?

(I don't really have anything against using keywords, but I prefer that using them add some sort of value -- e.g., they would be nice for the `owner` and `access` parameters which are otherwise anonymous in this context....)